src/muse/investments.py

```diff
@@ -346,7 +346,7 @@ def scipy_match_demand(
         )
         if not res.success:
             getLogger(__name__).critical(res.message)
-            print(f"in sector containing {df_technologies.technology[0]}")
+            print(f"in sector containing {df_technologies.technology.unique()}")
```

Comment:
We don't really want to be using `print` for error messages:

```suggestion
```

Alternatively you could include it in the logging message above if you think it's useful information.